yt_dlp/extractor/fosdem.py

```diff
@@ -0,0 +1,107 @@
+            'playlist_count': 11,
+            'info_dict': {
+                'id': 'microkernel_and_component_based_os',
+                'title': 'Microkernel and Component-based OS devroom',
+            }
+        }
+    ]
+
+    def _real_extract(self, url):
+        video_id = self._match_id(url)
+        groups = self._match_valid_url(url).groupdict()
+        webpage = self._download_webpage(url, video_id)
+        year = groups.get('year')
+        title_rgx = r'<div id=\"pagetitles\">\n\s+<h1>(.+?)</h1>'
+        title = self._html_search_regex(title_rgx, webpage, 'title') \
+            or self._og_search_title(webpage)
+        if groups.get('url_type') == 'event':
+            evnt_blurb_rgx = r'<div class=\"event-blurb\">\n*(?P<blurb>(<div class=\"event-abstract\">(<p>(.+?)</p>\n*)+</div>)+\n*(<div class=\"event-description\">(<p>(.+?)</p>\n*)*</div>))+\n*</div>'
+            evnt_blurb = self._html_search_regex(evnt_blurb_rgx,
+                                                 webpage,
+                                                 'event blurb',
+                                                 group='blurb',
+                                                 flags=re.DOTALL,
+                                                 fatal=False)
+            description = evnt_blurb
+            video_url_rgx = r'<li><a href=\"(https://video.fosdem.org/[0-9]{4}/.+)\">'
+            video_url = self._html_search_regex(video_url_rgx,
+                                                webpage,
+                                                'video url')
+            cast_rgx = r'<td><a href=\"/[0-9]+/schedule/speaker/[a-z_]+/\">(?P<speaker>\w+ \w+)</a></td>'
+            cast = re.findall(cast_rgx, webpage, flags=re.UNICODE) or []
+
+            return {
+                'id': video_id,
+                'title': title,
+                'description': description,
+                'uploader': 'FOSDEM',
+                'url': video_url,
+                'thumbnail': None,
+                'release_date': year,
+                'cast': cast,
+                'webpage_url': url,
+            }
+        elif groups.get('url_type') == 'track':
+            events_rgx = r'<td><a href=\"(?P<event>/[0-9]+/schedule/event/[a-z0-9]+/)'
+            events_slugs = re.findall(events_rgx, webpage) or []
+
+            if len(events_slugs) > 0:
+                events_urls = ['https://fosdem.org' + slug for slug in events_slugs]
+            entries = []
+            for event_url in events_urls:
+                entries.append(self.url_result(event_url, 'Fosdem'))
+            return self.playlist_result(entries,
+                                        playlist_id=video_id,
+                                        playlist_title=title,
+                                        playlist_description=None)
+        else:
+            url_type = groups.get('url_type')
+            print(f'The {url_type} is not supported')
```

Comment:
```suggestion
        return self.playlist_from_matches(
            re.findall(r'<td><a href="(?P<event>/[0-9]+/schedule/event/\w+/)', webpage),
            video_id, title, lambda x: f'https://fosdem.org{x}', ie=FosdomIE)
```

Reply:
is that `return` supposed to be in that `elif` or in an `else` block?